kinder/pkg/cluster/manager/actions/waiter.go

```diff
@@ -291,14 +291,16 @@ func staticPodHasVersion(pod, version string) func(c *status.Cluster, n *status.
 func kubeletHasRBAC(major, minor uint) func(c *status.Cluster, n *status.Node) bool {
 	return func(c *status.Cluster, n *status.Node) bool {
 		for i := 0; i < 5; i++ {
```

Comment:
should we increase the number of retries?
i think even 30 seconds here are fine.

Reply:
without any real clue on the underlying problem, I prefer to make the first try only improving the validation of the kubectl output...

BTW, this are not retries, but the number of consecutive times in a row the check should pass before assuming we are in a reliable state.

Reply:
i think you already explained this to me, but i again forgot.
i guess what i'm confused about is why the state can suddenly become unreliable.

once RBAC is applied it should remain there :thinking: